app/presenters/idv/by_mail/letter_enqueued_presenter.rb

```diff
@@ -24,21 +24,25 @@ def address_lines
       end
 
       def button_text
-        if sp
-          t('idv.cancel.actions.exit', app_name: APP_NAME)
-        else
-          t('idv.buttons.continue_plain')
-        end
+        t('idv.cancel.actions.exit', app_name: APP_NAME)
       end
```

Comment:
(Context: We settled on the button always having the same text, but keeping the destination dynamic.)